gc/gc-base/pom.xml

```diff
@@ -29,6 +29,11 @@
 
   <name>Nessie - GC - Base Implementation</name>
 
+  <properties>
+    <!--- We cannot use version > 0.43 beacuse it uses a kryo version 5.x that is incompatible with out spark (requires 4.x) -->
+    <kryo.serializers.version>0.43</kryo.serializers.version>
```

Comment:
should rather go into https://github.com/projectnessie/nessie/blob/main/pom.xml and the dependency + version should be defined in https://github.com/projectnessie/nessie/blob/main/pom.xml#L223. Then you can omit the `<version>${kryo.serializers.version}</version>` field when you define the dependency in this module here